sunpycube/cube/cube_utils.py

```diff
@@ -549,6 +549,9 @@ def get_cube_from_sequence(cubesequence, item):
                     data[i] = cube[item[2::]]
             result = cubesequence._new_instance(
                 data, meta=cubesequence.meta, common_axis=cubesequence.common_axis)
+        elif:
+            result = None
```

Comment:
Can delete this line setting result to None.  The next line raises an error anyway.